project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.11
+sbt.version=0.13.12
```

Comment:
This is not germane to the ticket, but is a good thing to do, and I'm hoping it jiggles this stubborn Travis failure.